nixos/modules/config/networking.nix

```
@@ -32,6 +32,22 @@ in
       '';
     };
 
+    networking.addFQDNAndHostnameEntries = lib.mkOption {
```

```suggestion
    networking.includeFQDNAndHostnameEntries = lib.mkOption {
```

This is a little bit of a nit, but since we're creating a new file every time and not rewriting something in place, we're defining what it contains -- not doing the "add" verb.